doc/source/conf.py

```diff
@@ -60,10 +59,10 @@
             "name": "Changelog",
         },
     ],
-    "switcher": {
-        "json_url": f"https://{cname}/release/versions.json",
-        "version_match": get_version_match(__version__),
-    },
+    # "switcher": {
+    #    "json_url": f"https://{cname}/release/versions.json",
+    #    "version_match": get_version_match(__version__),
+    # },
```

Comment:
Do not leave commented out code. If it is no longer needed, delete it.
```suggestion
```

Reply:
Yes, i was testing it, to check it is failing or not.